pkg/hs/urbit-king/lib/Urbit/Vere/Pier.hs

```diff
@@ -360,6 +361,29 @@ pier (serf, log) vSlog startedSig = do
   tExec <- acquireWorker "Effects" (router slog (readTQueue executeQ) drivz)
   tDisk <- acquireWorkerBound "Persist" (runPersist log persistQ execute)
 
+  -- Now that the Serf is configurated, the IO drivers are hooked up, their
+  -- starting events have been dispatched, and the terminal is live, we can now
+  -- handle injecting events requested from the command line.
+  unless (null injected) $ rio $ do
+    for_ (zip [1..] injected) $ \(num, ev) -> do
```

Comment:
I think the unless here is unnecessary. If the list is empty, `for` won't do anything.